src/Command/ModelCommand.php

```diff
@@ -127,9 +127,12 @@ public function bake(string $name, Arguments $args, ConsoleIo $io): void
     public function validateNames(TableSchemaInterface $schema, ConsoleIo $io): void
     {
         foreach ($schema->columns() as $column) {
-            if (!is_string($column) || (!ctype_alpha($column[0]) && $column[0] !== '_')) {
+            $re = '/^[a-zA-Z_][a-zA-Z0-9_]*$/m';
```

Comment:
Why do we need the `m` modifier?

Reply:
right, we don't need multiline here so I will remove it